pkg/storage/azureblob/getter.go

```diff
@@ -0,0 +1,85 @@
+package azureblob
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"github.com/gomods/athens/pkg/config"
+	"github.com/gomods/athens/pkg/errors"
+	"github.com/gomods/athens/pkg/observ"
+)
+
+// Info implements the (./pkg/storage).Getter interface
+func (s *Storage) Info(ctx context.Context, module string, version string) ([]byte, error) {
+	const op errors.Op = "azureblob.Info"
+	ctx, span := observ.StartSpan(ctx, op.String())
+	defer span.End()
+
+	exists, err := s.Exists(ctx, module, version)
+	if err != nil || !exists {
+		return nil, errors.E(op, errors.M(module), errors.V(version), errors.KindNotFound)
```

Comment:
we should not swallow errors, errors are valuable for investigating issues in prod, you can inject it like this
`return nil, errors.E(op, errors.M(module), errors.V(version), errors.KindNotFound, err)`

Reply:
same for everywhere

Reply:
err will be nil in that case so no need to do that

Reply:
it enters the block on err != nil or !exists so it may or may not be nil

Reply:
@michalpristas I think the "Conversation" Tab shows different code that the "Files Changed" Tab b/c here i see
```
exists, err := s.Exists(ctx, module, version)
if err != nil {
	return nil, errors.E(op, err, errors.M(module), errors.V(version))
}
if !exists {
    return nil, errors.E(op, errors.M(module), errors.V(version), errors.KindNotFound)
}
```